user/pages/03.commerce2/02.developer-guide/07.orders/03.customize-order-receipt-emails/docs.md

```diff
@@ -17,7 +17,27 @@ The order receipt email, which customers receive, is controlled through the `com
 The default `commerce-order-receipt.html.twig` can be found in the `commerce_order` module. 
 
 The email can be customized by overriding the Twig template into your theme.
+ 
+Or in your custom module:
```

Comment:
Should we tell people how to copy into theme? maybe even link to Drupal.org docs for these things?

Reply:
ya probably, i assumed it was omitted bcuz its so common.